structure/extension/_ID.yaml

```diff
@@ -0,0 +1,30 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://github.com/glamberson/gedcom-evidence/_ID
+
+extension tags:
+  - _ID
+
+specification:
+  - Evidence Identifier
+  - |
+    An identifier reference from evidence to an individual, family, or other record. This creates a bidirectional link between evidence and conclusions, allowing evidence to be associated with multiple potential individuals. The _ID structure supports gradual refinement of identity as research progresses, with confidence levels and research documentation.
+
+label: Evidence Identifier
+
+payload: "@<https://gedcom.io/terms/v7/record-INDI>@ | @<https://gedcom.io/terms/v7/record-FAM>@ | @<https://gedcom.io/terms/v7/record-SOUR>@"
+
+substructures:
+  "https://gedcom.io/terms/v7/NOTE": "{0:M}"
+  "https://gedcom.io/terms/v7/QUAY": "{0:1}"
+  "https://github.com/glamberson/gedcom-evidence/_RDOC": "{0:M}"
```

Comment:
In https://github.com/glamberson/gedcom-evidence/blob/master/specification.md the _ID structure is listed with a text payload and no substructures. I'm curious which one is more correct.